zulipterminal/ui_tools/boxes.py

```diff
@@ -314,7 +317,7 @@ def _is_private_message_to_self(self) -> bool:
                 and recipient_list[0]['email'] == self.model.user_email)
 
     def stream_header(self) -> Any:
-        stream_topic_separator = '▶'
+        stream_topic_separator = STREAM_TOPIC_SEPARATOR
```

Comment:
There seems to be only one occurance of `stream_topic_separator` further below, so we could just replace that with `STREAM_TOPIC_SEPARATOR` instead of doing this substitution.